sel4test-hw/build.py

```diff
@@ -89,6 +89,11 @@ def build_filter(build: Build) -> bool:
         if build.is_hyp() and (build.is_smp() or build.is_verification()):
             return False
 
+    if plat.arch == 'riscv':
```

Comment:
Let's make this platform specific and not arch generic? At least we should see them how the Star64 board works once it's in.